portal-web/test/functional/com/liferay/portalweb/tests/MyTestsBlogs/MyBlogTests.testcase

```diff
@@ -0,0 +1,22 @@
+	setUp {
```

Comment:
Tu precisaria criar uma pagina aqui tambem e adicionar o blog a ela 
`Given the Blogs Portlet is added to the page`